Makefile

```diff
@@ -60,17 +60,15 @@ BIN_DIR = $(CURDIR)/build/_output/bin/
 export GOPROXY=direct
 export GOSUMDB=off
 export GOFLAGS=-mod=vendor
-export GOROOT=$(BIN_DIR)/go/
-export GOBIN=$(GOROOT)/bin/
-export PATH := $(GOROOT)/bin:$(PATH)
 
 export KUBECONFIG ?= $(shell ./cluster/kubeconfig.sh)
 export SSH ?= ./cluster/ssh.sh
 export KUBECTL ?= ./cluster/kubectl.sh
 
+GO_VERSION = $(shell grep "^go " go.mod |awk '{print $$2}')
```

Comment:
typo: $$2 -> $2

Reply:
At Makefiles it has to be `$$` to escape `$`